pulsar-client/src/test/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImplTest.java

```diff
@@ -62,4 +65,23 @@ public void testGetStats() throws Exception {
         impl.getStats();
     }
 
+    @Test
+    public void multiTopics() throws Exception {
```

Comment:
How about rename it to multiTopicsInDifferentNameSpace, and check that messages send from different topic could receive successfully by this consumer?

Reply:
I have fixed by comments，PTAL.